### PR TITLE
docs: Added information about scopes (#2782)

### LINKED ISSUE #2782: Docs SSO RBAC Scopes Param
Checklist:

* [x] I've searched in the docs and FAQ for my answer: http://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

https://github.com/argoproj/argo-cd/blob/bbfb96cb0161b1d11cbbc334fb33eb1158b2c075/docs/operator-manual/argocd-rbac-cm.yaml#L27-L29
scopes Param from example rbac configmap is not mentioned in docs. I was debugging our Okta groups not getting mapped to policies and noticed that without setting **scopes : '[Groups]'** the group mappings would fail. The issue itself maybe related to our Okta Settings but I thought it would be good to let people know that this may be necessary. Interestingly, after checking with Okta Team, I was told that the scope indeed got created with the value **groups** not **Groups**. 

**To Reproduce**

Docs Issue

**Expected behavior**

Docs be precise

**Version**
Current Master

### PR DESCRIPTION
**Pull request by christianh814**
Added some more information about the `scopes` field and what will the default be if the user doesn't provide anything. Fixes #2782

### CODE REVIEW

File: docs/operator-manual/rbac.md [hunk 1]
```diff
@@ -171,6 +171,32 @@ g, db-admins, role:staging-db-admins
...
+  namespace: argocd
+  labels:
+    app.kubernetes.io/name: argocd-rbac-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  policy.csv: |
+    p, my-org:team-alpha, applications, sync, my-project/*, allow
+    g, my-org:team-beta, role:admin
+  policy.default: role:readonly
+  scopes: '[groups, email]'
```
Comment (crenshaw-dev) [inline docs/operator-manual/rbac.md:195]: Should the example also have a `g, ` line targeting an example email address?

File: docs/operator-manual/rbac.md [hunk 2]
```diff
@@ -171,6 +171,32 @@ g, db-admins, role:staging-db-admins
...
+metadata:
+  name: argocd-rbac-cm
+  namespace: argocd
+  labels:
+    app.kubernetes.io/name: argocd-rbac-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  policy.csv: |
+    p, my-org:team-alpha, applications, sync, my-project/*, allow
+    g, my-org:team-beta, role:admin
```
Comment (todaywasawesome) [inline docs/operator-manual/rbac.md:193]:
```suggestion
    g, my-org:team-beta, role:admin
    g, user@example.org, role:admin
```

Like this @crenshaw-dev ?

File: docs/operator-manual/rbac.md (hunk 1 above)
Comment (christianh814) [reply docs/operator-manual/rbac.md:196]: Yeah, good idea! Added suggestion from @todaywasawesome

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**